code/modules/research/techweb/all_nodes.dm

```diff
@@ -677,7 +677,20 @@
 	display_name = "IPC Parts"
```

Comment:
Strangely this is a tier 3 tech, but it has a tier 4 requirement.

Reply:
Changed to tier 4